src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java

```diff
@@ -233,4 +240,38 @@ public CaseDetails submitUpdate(String authorization, String caseId,
             caseDataContent
         );
     }
+
+    /**
+     * Given a list of caseIds, this method will return a list of case details.
+     *
+     * @param authorisation used for IDAM authentication for the query
+     * @param caseIds       used as the query parameter
+     * @return a list of case details
+     */
+    public List<CaseDetails> getCaseData(String authorisation, List<String> caseIds) {
+        BoolQueryBuilder boolQueryBuilder = boolQuery()
+            .filter(new TermsQueryBuilder("reference.keyword", caseIds));
+        String query = new SearchSourceBuilder()
+            .query(boolQueryBuilder)
+            .toString();
+
+        return searchEnglandScotlandCases(authorisation, query);
+    }
+
+    private List<CaseDetails> searchEnglandScotlandCases(String authorisation, String query) {
+        List<CaseDetails> caseDetailsList = new ArrayList<>();
+        caseDetailsList.addAll(searchCaseType(authorisation, ENGLAND_CASE_TYPE, query));
+        caseDetailsList.addAll(searchCaseType(authorisation, SCOTLAND_CASE_TYPE, query));
+        return caseDetailsList;
```

Comment:
Minor: Could have simplified...
```
return List.of( searchCaseType(authorisation, ENGLAND_CASE_TYPE, query),
    searchCaseType(authorisation, SCOTLAND_CASE_TYPE, query) );
```

Reply:
Noted - change can go in either the 2nd or 3rd API PR